README.md

```diff
@@ -284,25 +284,31 @@ Many examples of `Module` lifecycle behavior and manipulation can be found in th
 
 ### Lifecycle Methods
 
-`Module` exposes just three lifecycle methods that external consumers should use to trigger loading and unloading
+`Module` exposes five lifecycle methods that external consumers should use to trigger loading and unloading
 behavior:
 
 Method         | Description
 -------------- | ---------------------------------
 `load`         | Triggers loading of a `Module`.  Internally, this executes the module's `onLoad` method and dispatches the `willLoad` and `didLoad` events.  Returns a future that completes once the module has finished loading.
 `shouldUnload` | Returns the unloadable state of the `Module` as a `ShouldUnloadResult`.  Internally, this executes the module's `onShouldUnload` method.
 `unload`       | Triggers unloading of a `Module`.  Internally, this executes the module's `shouldUnload` method, and, if that completes successfully, executes the module's `onUnload` method. If unloading is rejected, this method will complete with an error.
```

Comment:
`unload` will unload all child modules too. `shouldUnload` will check the state of all child modules as well. It might be worth updating the descriptions of these two methods to note that.